ge0/parser.cpp

```diff
@@ -50,6 +50,32 @@ bool Ge0Parser::Parse(std::string const & url, Result & result)
 
 bool Ge0Parser::ParseAfterPrefix(std::string const & url, size_t from, Result & result)
 {
+  std::string remaining = url.substr(from);
```

Comment:
std::string_view avoids creating copies of the original string, while uses the same string api.